yarn.lock

```diff
@@ -14389,7 +14394,20 @@ mongodb-memory-server@6.6.7:
   dependencies:
     mongodb-memory-server-core "6.6.7"
 
-mongodb@3.6.8, mongodb@3.6.9, mongodb@^3.5.9, mongodb@^3.6.4:
+mongodb@3.6.8:
```

Comment:
We've got two versions of MongoDB installed (3.6.8 and 3.6.9). Shall we have just one?